exasol_bucketfs_utils_python/buckets.py

```diff
@@ -0,0 +1,36 @@
+from exasol_bucketfs_utils_python import BucketFsError
+import requests
+from requests.auth import HTTPBasicAuth
+from typing import Generator
+
+
+def list_buckets(
+    base_url: str,
```

Comment:
Another thing, all other functions use the config objects only this uses the parameter directly.

Reply:
Well all the others api calls a `BucketConfig` and iteract with a specific bucket rather than with the `BucketFS` service.

Reply:
There is a BucketFSConfig which is part of the BucketConfig

Reply:
Maybe we can also directly return BucketConfig's

Reply:
Do you mean the `BucketFSConnectionConfig`? From my point of view this just makes the api unnecessarily complex.